x/deployment/handler/handler.go

```diff
@@ -36,10 +36,6 @@ func handleMsgCreate(ctx sdk.Context, keeper keeper.Keeper, _ MarketKeeper, msg
 		// Version: sdk.Address.Bytes(),
 	}
 
-	if _, found := keeper.GetDeployment(ctx, deployment.ID()); found {
-		return nil, types.ErrDeploymentExists
-	}
```

Comment:
should we `panic()` here or in `keeper.Create()`?

Reply:
`keeper.Create()` looks as right place